### PR TITLE
docs: Match open bracket with close bracket in unsqueeze

### PR DESCRIPTION
**Pull request by connor-henderson**
Was going to fix something else that I thought was an issue, but isn't, so just leaving this tiny thing in case it's wanted

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

**Review by kit1980 — APPROVED** [review]

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -12051,7 +12051,7 @@ def merge_dicts(*dicts):
 
 The returned tensor shares the same underlying data with this tensor.
 
-A :attr:`dim` value within the range ``[-input.dim() - 1, input.dim() + 1)``
+A :attr:`dim` value within the range ``[-input.dim() - 1, input.dim() + 1]``
```
Comment (albanD) [inline torch/_torch_docs.py:12054]: This change is not ok!
`input.dim() + 1` is not part of the valid range and that's why `)` is used instead of `]`.

Comment (albanD) [reply torch/_torch_docs.py:12054]: If this convention is not clear for all readers (it is a norm in european math but might not be universal). We should remove the `+1` on the second number: `[-input.dim() - 1, input.dim()]`

**Comment by huydhn** [comment]
@pytorchbot drci

(testing the bot)

**Pull request closed without merging**